docs-ref-conceptual/msal-based-azure-cli.md

```diff
@@ -34,18 +34,45 @@ Alternatives you may consider for stability:
 
 ### Calling `az account get-access-token`
 
-You can manually call [`az account get-access-token`](/cli/azure/account#az_account_get_access_token) in a terminal or use subprocess to call it from another programming language. By default, the returned access token is for Azure Resource Manager (ARM) and the default subscription/tenant shown in [`az account show`](/cli/azure/account#az_account_show).
+You can manually call [`az account get-access-token`](/cli/azure/account#az_account_get_access_token) in a terminal or use a subprocess to call it from another programming language. By default, the returned access token is for Azure Resource Manager (ARM) and the default subscription/tenant shown in [`az account show`](/cli/azure/account#az_account_show). 
 
-```azurecli
-# get the active subscription
+Starting from Azure CLI 2.54.0, `az account get-access-token` returns an additional property which makes it supported by MSAL-based Azure CLI. See the following for an example of the access token output of the changes:
```

Comment:
This sentence about access tokens is inside a code block to get the active subscription?  Maybe GitHub preview is a little confused.